### ISSUE TITLE
Inverted favicons for dark themes

### ISSUE THREAD

**Issue by jiri-h** [Croydon/vertical-tabs-reloaded#247]
In version 0.13.0 there is a new feature that inverts color of favicons for dark themes. I think this was unnecessary as for me at least the icon visibility was fine. The problem now is, that some icons have now a different color than what I am used to. For example Evernote - I am looking for green icon, now it is pink. Dropbox - I am looking for blue, but it is orange... I can't now navigate subconsiously and I have to actively look for the icon.

It would be great to have a checkbox in the extension settings, something like "Invert color of favicons (suitable for dark themes)" that would be optional, not mandatory.

Anyway, I appreciate very much the work on this extension, it is an essential one for me and I like it very much.

Best regards,
Jiri

**Comment by akunaatrium** [comment]
Yes, agreed. This "feature" should definitely be removed or at least make it an option. The original icons looked just fine on a darker background.

**Comment by nullbasis** [comment]
Same problem here. If the feature stays, can it at least be made a little smarter? I don't mind black favicons turning white, but red ones turning green is a problem for me.
